hoot-core/src/main/cpp/hoot/core/info/ApiEntityDisplayer.cpp

```diff
@@ -114,44 +119,25 @@ void printApiEntities(const std::string& apiEntityBaseClassName, const QString a
       }
       const int indentAfterName = maxNameSize - name.size();
       const int indentAfterType = MAX_TYPE_SIZE - apiEntityType.size();
-      QString line = "  " + name + QString(indentAfterName, ' ');
+      QString line_buffer;
+      QTextStream line(&line_buffer);
+      line << "  " << name << QString(indentAfterName, ' ');
       if (displayType)
       {
-        line += apiEntityType + QString(indentAfterType, ' ');
+        line << apiEntityType << QString(indentAfterType, ' ');
       }
-      line += apiEntityInfo->getDescription();
-      LOG_VARD(line);
-      std::cout << line << std::endl;
+      line << apiEntityInfo->getDescription();
+      LOG_VARD(line.readAll());
+      ts << line.readAll() << endl;
     }
   }
-  std::cout << std::endl;
-}
-
-QString ApiEntityDisplayer::_apiEntityTypeForBaseClass(const QString className)
-{
```

Comment:
This function just got moved down further in the source file (unchanged) to keep the two template, non-member functions together.